feat/gene/gene.go

```diff
@@ -220,39 +220,43 @@ func (t *CodingTranscript) Orientation() feat.Orientation {
 
 // UTR5start returns the start of the 5'UTR relative to the transcript.
 func (t *CodingTranscript) UTR5start() int {
-	if t.Orient == feat.Forward {
+	ori, _ := feat.BaseOrientationOf(t)
+	if ori == feat.Forward {
 		return 0
-	} else if t.Orient == feat.Reverse {
+	} else if ori == feat.Reverse {
 		return t.CDSend
```

Comment:
Mmmmm. I really don't like that the positions are relative to the transcript. There needs to be some way around that; maybe add `UTR5() feat.Feature` and `UTR3() feat.Feature` which can then be passed to `BasePositionOf`. This can be a future PR.